UPGRADE_GUIDE.md

```diff
@@ -1,5 +1,78 @@
 # Upgrade Guide
 
+## From `0.2.6` to `0.2.7
+
+### Worker SDK `14.0.0` upgrade
+
+There are a number of breaking changes to contend with for the Worker SDK upgrade:
+
+#### Removal of `Vector3f` and `Vector3d`
+
+These two schema types are no longer available in the schema standard library. You can replace their definitions by defining a schema file in your project:
+
+```
+package my_game;
+
+type Vector3f {
+    float x = 1;
+    float y = 2;
+    float z = 3;
+}
+
+type Vector3d {
+    double x = 1;
+    double y = 2;
+    double z = 3;
+}
+```
+
+And replacing the import of `improbable/vector.schema` and usage of `improbable.Vector3f`/`improbable.Vector3d` with the schema file you defined.
+
+> Note that methods such as `Vector3f.ToUnityVector();` are no longer available and you'll need to reimplement them yourself as extension/static methods. You can find the old implementations here: [`Vector3f`](https://github.com/spatialos/gdk-for-unity/blob/0.2.6/workers/unity/Packages/io.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Partials/Improbable.Vector3f) and [`Vector3d`](https://github.com/spatialos/gdk-for-unity/blob/0.2.6/workers/unity/Packages/io.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Partials/Improbable.Vector3d).
+> 
+> You will be unable to reimplement the operators since C# lacks the ability to define operations via extension methods.
```

Comment:
could suggest that `Vector3d` is basically the same structure as `Coordinates`?

Reply:
Added a small sentence